lib/components/molecules/Pagination/PaginationList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '../../atoms/Button';
+
+const PaginationList = ({ pageNumbers, executePagination, activePage, currentPageAria }) => {
+  return (
+    pageNumbers &&
+    pageNumbers.map(page => {
+      return (
+        <>
+          <li key={page} className="page-item">
+            <Button
+              aria-label={`${currentPageAria} ${activePage}`}
```

Comment:
We need to fix the aria content and indicator for current and active buttons distinctively .